### PR TITLE
Skgrid minor

### PR DESCRIPTION
**Pull request by jordan2lee**
skgrid will train models from an algorithm found in it's classifier config file (cleaner, can point to a file for the skgrid model lib options easily, etc)

streamline dockerfile and add new classifier config file

### CODE REVIEW

File: skgrid/Dockerfile [hunk 1]
```diff
@@ -6,10 +6,7 @@ WORKDIR /skgrid/
...
 COPY data ./data
 
-COPY set_model_params.py create_grid.py mlib.sh run_train.py train.sh predict.sh prediction_runner.py pick_model.py select_model.yml ./
+COPY config_classifier.json set_model_params.py create_grid.py mlib.sh run_train.py train.sh predict.sh prediction_runner.py pick_model.py select_model.yml ./
 RUN chmod -R o+rw ./
-RUN chmod 777 ./mlib.sh
-RUN chmod 777 ./predict.sh
-RUN chmod 777 ./train.sh
-RUN chmod 777 ./run_train.py
+RUN chmod 777 ./mlib.sh ./predict.sh ./train.sh ./run_train.py
```
Comment (kellrott) [inline skgrid/Dockerfile:11]: Why does this need to be done in the dockerfile?

File: skgrid/run_train.py [hunk 2]
```diff
@@ -31,41 +31,12 @@
...
-    "sklearn.gaussian_process.kernels.RationalQuadratic": RationalQuadratic,
-    "sklearn.gaussian_process.kernels.WhiteKernel": WhiteKernel
-}
-
-
 def get_classifier(name, params):
     if name == "sklearn.gaussian_process.GaussianProcessClassifier":
         d = copy(params)
-        d['kernel'] = gp_kernels[params['kernel']]()
+        d['kernel'] = eval(classifier_config['gp_kernels'][d['kernel']])()
```
Comment (kellrott) [inline skgrid/run_train.py:37]: A bare 'eval' on a config file is considered a bad security process

File: skgrid/run_train.py [hunk 3]
```diff
@@ -76,19 +47,14 @@ def get_classifier(name, params):
...
-    #     except yaml.YAMLError as exc:
-    #         print(exc)
-    # model_file = modelconfig[args.cancer][args.platform]['file']
-
     configs = []
     with open(args.model_file) as handle:
         for line in handle:
             configs.append( json.loads(line) )
 
+    with open("/skgrid/config_classifier.json", 'r') as fh:
```
Comment (kellrott) [inline skgrid/run_train.py:55]: Why is this not configurable?

File: skgrid/run_train.py [hunk 4]
```diff
@@ -31,41 +31,12 @@
 from sklearn.gaussian_process.kernels import RationalQuadratic
 from sklearn.gaussian_process.kernels import WhiteKernel
 
-
-classifers = {
```
Comment (kellrott) [inline skgrid/run_train.py:35]: What is the reasoning for moving this code to a config file? These link to the actual classes, which can't be serialized to JSON

**Review by kellrott — CHANGES_REQUESTED** [review]

**Comment by jordan2lee** [comment]
closing because better method is to keep classifier info in the source file rather than separate it into a sep config file

**Pull request closed without merging**